src/index.js

```diff
@@ -1,10 +1,25 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { render } from 'react-dom';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import './index.css';
 import App from './App';
+import { Nav } from './global/Nav';
+import { AddItems } from './components/Add-items/Add-items';
+import { List } from './components/List/List';
```

Comment:
Suggestion (and personal best practice):

In some cases, it's confused when we mix "components" and "components as pages" (because at React "everything" could be considered as a component). I usually create two folder: `components` and `pages` for a lot of reasons. For example organization code, easy to review and maintain, and separate concerns. 😄 😄